web3/types.py

```diff
@@ -177,7 +177,7 @@ class LogReceipt(TypedDict):
     address: ChecksumAddress
     blockHash: HexBytes
     blockNumber: BlockNumber
-    data: HexStr
+    data: HexBytes
     logIndex: int
     payload: HexBytes
```

Comment:
Should `payload` and `topic` be removed?

Reply:
I think you are correct. I can't find any reference to expected `payload` or `topic` fields, not sure where those came from. Go ahead and remove, thank you!